spec/presenters/presentation_presenter_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe PresentationPresenter do
+  subject { described_class.new(presentation, nil) }
+
+  describe '#associable_events' do
```

Comment:
That makes an assumption about the internal implementation of this method. Really I dont care if you use the upcoming scope or some other mechanism for filtering Events.

But what I care about testing is behaviour: 
- No past events are included 
- The currently associated event is present in the list